src/physics/cam/yog_conv_intr.F90

```diff
@@ -0,0 +1,923 @@
+module yog_conv_intr
```

Comment:
This is a big file with some very long subroutines. Can it be modularised/subdivided further, to make it easier to read and adapt?